pydatarecognition/cif_io.py

```diff
@@ -222,6 +222,21 @@ def cif_read_ext(cif_file_path, client):
     return None
 
 
+def cif_read_mpcontribs(cif_type: str) -> str:
+    """
+    given the type of cif files, returns 'Not implemented'
```

Comment:
The first line is a high level description of the what the function does.  This is not clear to me from what you wrote

If this function is just handling the for loop and not anything else, I would say something like
`returns not implemented warning if --mpcontribs_client is selected at the command line`

I would have it take a bool (i.e., the value of `mpcontribs_client` which is `True` or `False`).  Then we have to decide what it returns. If it returns a string that is the error message then we have to have lots of code on the other side to print that, also what string does it return if `mpcontribs` is `False`?  We want it to print nothing in that case for backwards compatibility.  Probably it should return nothing (which is the same as returning `None` in Python I think.

This is why docstrings are so useful....figuring out what the function will do!

Now the name of the function is better if it makes your code readable.  I think something like `set_client(args.mpcontribs_client)` because the name is more saying what the function does (function names are better as verbs).

This raises another question about what the arg could be.  We could have an arg that is `--client` and it can take values `mpcontribs`, `mpc`, or `fs` and is by default `fs`.  Maybe this iwhat you had in mind and is why you wanted to pass in a string.  In that case, ignore the things I said above.

Anyway, great start.  This is what I mwan about the value of writing docstrings